source/sonata_simulation.rst

```diff
@@ -96,6 +96,27 @@ example::
        "forward_skip": 500
   }
 
+LFPWeightsPath
+--------------
+
+*Optional*
+A parameter specifying the path to the weights file describing the scaling factors for the contributions of each segment's transmembrane current to the LFP/EEG. The format of the weights file is as follows:
+
+.. table::
+  =========================== ======================= ========== ============= =============================================================================================================================================================================================================
+  Group                       Field                   Type       Requirement   Description
+  =========================== ======================= ========== ============= =============================================================================================================================================================================================================
+  /electrodes/{electrodename} layer                   str        Optional      Layer of the circuit in which {electrodename} is located
+  /electrodes/{electrodename} location                float64    Optional      Position of {electrodename} in cartesian coordinates (3 x 1)
+  /electrodes/{electrodename} region                  str        Optional      Region in which {electrodename} is located
+  /electrodes/{electrodename} type                    str        Optional      Either EEG or LFP
+  /electrodes/{electrodename} offset                  uint64     Optional      Column index of the coefficients for {electrodename} in /electrodes/electrode_grid/{gid}
+  /electrodes/electrode_grid  {gid}                   float64    Mandatory     Scaling factor for each segment in the corresponding neuron (nsegments x nelectrodes)
+  /                           neuron_ids              uint64     Mandatory     List of GIDs (ngids x 1)
```

Comment:
we use population, node_ids in SONATA to identify cells. so this this to be adjusted.